plugins/Model_IAE/AutoEncoder.py

```diff
@@ -39,11 +38,6 @@ def load(self, swapped):
             return False
 
     def save_weights(self):
-        model_dir = str(self.model_dir)
```

Comment:
Maybe I'm misreading this, but it looks like it's pulling out this code, which was just merged in. If I'm not misreading, then I think it's just some over-zealous conflict resolution ;)